spec/json_api_filter_spec.rb

```diff
@@ -103,9 +105,90 @@ class FakesController
                 name: {eq: "foo"}
               }
             },
-            request: User.where("id <= 1").where("name = 'foo'")
+            request: User.where("id <= 1").where("name" => "foo")
           }
        ]
+      },
+      {
+        name: "JsonApiFilter::FieldFilters::Searcher",
+        examples: [
+          {
+            name: "global search",
+            params: {
+              search: "test user"
+            },
+            request: User.where("name = 'test user'")
+          },
+          {
+            name: "column search",
+            params: {
+              filter: {
+                name: {
+                  search: "test user"
+                }
+              }
+            },
+            request: User.where("name = 'test user'")
+          }
+        ]
+      },
+      {
+        name: "JsonApiFilter::FieldFilters::Sorter",
+        examples: [
+          {
+            name: "sort by id",
+            params: {
+              sort: {
+                by: "id"
+              }
+            },
+            request: User.order("id")
+          },
+          {
+            name: "sort by descending id",
+            params: {
+              sort: {
+                by: "id",
+                desc: true
+              }
+            },
+            request: User.order("id").reverse_order
+          },
+          {
+            name: "sort by name",
+            params: {
+              sort: {
+                by: "name"
+              }
+            },
+            request: User.order("name")
```

Comment:
related to sorter.rb suggestion
```suggestion
            request: User.order(name: :asc)
```